pyvista/utilities/geometric_objects.py

```diff
@@ -692,7 +692,16 @@ def CircularArc(pointa, pointb, center, resolution=100, normal=None,
             arc.SetAngle(angle)
 
     arc.Update()
-    return pyvista.wrap(arc.GetOutput())
+    arc = pyvista.wrap(arc.GetOutput())
+    # Compute distance of every point along circular arc
+    pointa = np.array(pointa)
+    center = np.array(center)
+    points = arc.points
+    radius = np.sqrt(np.sum((pointa-center)**2, axis=0))
+    chords = np.sqrt(np.sum((points-pointa)**2, axis=1))
+    thetas = 2.0*np.arcsin(chords/(2.0*radius))
+    arc['Distance'] = radius*thetas
```

Comment:
This returns NAN for certain arcs.  Specifically, in our unit test:
https://github.com/pyvista/pyvista/blob/07d2fdd0ae655ae3aecf477c84098929effbb4f2/tests/test_geometric_objects.py#L150-L151

We should add a test to verify that we compute the distance along an arc correctly.

Reply:
Thanks. I resolve it.